gz-resources.html

```diff
@@ -468,6 +468,10 @@
           }
         }
         console.error(msg)
```

Comment:
I actually thought about that and decided that the original msg is nicer because the way the browser shows objects (with arrows you can expand and use in the console)